transport/stunnel/stunnel.go

```diff
@@ -290,3 +321,18 @@ func markForCleanup(ctx context.Context, c ctrlclient.Client, objKey types.Names
 
 	return nil
 }
+
+// GeneratePassword can be used to generate random character string for 24 byte
```

Comment:
24 --> 32 bytes?

Reply:
@JohnStrunk yeah as per Stunnel documentation, stunnel allows either 16 or 32 bytes PSK. I didn't really try giving a 24 key value, I can check if its something we want to do.

Reply:
No problem. The comment just says 24, but the implementation is 32.

Reply:
ahh okay.